src/transaction.cpp

```diff
@@ -1967,7 +1975,11 @@ void wsrep::transaction::streaming_rollback(
             client_state_.server_state_.convert_streaming_client_to_applier(
                 &client_state_);
             lock.lock();
-            streaming_context_.cleanup();
+            // Postpone streaming_context cleanup to after_rollback().
```

Comment:
I find it a bit weird to leave a comment for a line of code that has been removed.
Perhaps it would make more sense to leave a general comment? To remind that streaming_rollback() may be called from bf_abort() while the "owning thread" may be executing (with its transaction object unlocked!). So we don't want to unnecessarily mess with the state of this transaction.